openhands/runtime/plugins/agent_skills/file_ops/file_ops.py

```diff
@@ -115,10 +124,14 @@ def _print_window(file_path, targeted_line, window, return_str=False):
         # cover edge cases
         CURRENT_LINE = _clamp(targeted_line, 1, total_lines)
         half_window = max(1, window // 2)
-
-        # Ensure at least one line above and below the targeted line
-        start = max(1, CURRENT_LINE - half_window)
-        end = min(total_lines, CURRENT_LINE + half_window)
+        if ignore_window:
+            # Use CURRENT_LINE as starting line (for e.g. scroll_down)
```

Comment:
nice improvement!!

Reply:
But, if we are scrolling up, shouldn't the CURRENT_LINE be the last line? if that make sense?

Reply:
Oh, good catch, I'll look into this asap!